### PR TITLE
Outreach Partners blog

### PR DESCRIPTION
**Pull request by Sc0rpi0n101**
I've added the blog for Outreach Partners. 

- Is there anything that needs to be added/changed?
- Still need a contact link here. 
- Also, the About Us page needs to be deployed before merging this, as that link is currently not working. 

CC: @vipulgupta2048 @abhishekmishragithub

### CODE REVIEW

File: content/blog/2020-outreach-partners.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Title: Bringing PyCon India to a wider audience with your help
+Date: 2020-07-23 13:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-outreach-partners
+
+[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about the conference in their communities and help bring more fresh faces to the event.
```
Comment (vipulgupta2048) [inline content/blog/2020-outreach-partners.md:7]:
```suggestion
[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about our conference in communities all over the world and bring fresh faces to the platform in hopes of creating a more inclusive, accessible platform this year.
```

Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:7]: Can we use this but break it down into 2 sentences. If that helps

File: content/blog/2020-outreach-partners.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Date: 2020-07-23 13:00
+Author: Nikhil Maan (@sc0rpi0n101)
+Category: 2020
+Slug: 2020-outreach-partners
+
+[**PyCon India**](https://in.pycon.org/2020/) is looking for outreach partners that can help spread the word about the conference in their communities and help bring more fresh faces to the event.
+
+<!-- PELICAN_END_SUMMARY -->
+
+We are the largest gathering of Python users and admirers in India who come together once a year to celebrate the language and share our experience and expertise with the community.
```
Comment (vipulgupta2048) [inline content/blog/2020-outreach-partners.md:11]:
```suggestion
PyCon India is the largest gathering of Python learners, programmers and enthusiasts in India who come from all over the world together once a year to celebrate the programming language and share their experiences, insight and expertise with the community. Last year alone we had an audience of 3000+, with over 300+ proposals submitted and a packed 2-day agenda in addition to 50+ sponsors supporting us. This year we like to bring forward positive change, and make the conference even more inclusive by bringing forth combined efforts of our communities in helping PyCon India reach more viewers.
```

Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:11]: @anistark @pythonindia/pycon-india-2020 Can someone confirm these stats are correct?

File: content/blog/2020-outreach-partners.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+Let's make make PyCon India a great conference using the combined effort of our communities.
```
Comment (vipulgupta2048) [inline content/blog/2020-outreach-partners.md:13]: If the above suggestion, looks good. We won't need this line.

File: content/blog/2020-outreach-partners.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+<!-- PELICAN_END_SUMMARY -->
+
+We are the largest gathering of Python users and admirers in India who come together once a year to celebrate the language and share our experience and expertise with the community.
+
+Let's make make PyCon India a great conference using the combined effort of our communities. 
+
+## What we expect from you 
+
+We would love for you to spread a word about the conference among your members who are looking to learn something regarding Python. We expect our outreach partners to help us in the following ways:
```
Comment (vipulgupta2048) [inline content/blog/2020-outreach-partners.md:17]: Start with "We would like to invite your community to be a Outreach partner for PyCon India 2020"

Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:17]: Then ease into expectations.

File: content/blog/2020-outreach-partners.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+We are the largest gathering of Python users and admirers in India who come together once a year to celebrate the language and share our experience and expertise with the community.
+
+Let's make make PyCon India a great conference using the combined effort of our communities. 
+
+## What we expect from you 
+
+We would love for you to spread a word about the conference among your members who are looking to learn something regarding Python. We expect our outreach partners to help us in the following ways: 
+
+* Send emails to your mailing lists and post content on their social media channels on our behalf to make your community members aware of the conference.
```
Comment (vipulgupta2048) [inline content/blog/2020-outreach-partners.md:19]: Both 2nd and 3rd person being used here. Pick one either go with `your` or `their`

Rather emails or being specific, convert it to "Share content on mailing list, social media channels, and messaging groups with your community members about the conference and all its updates,"

File: content/blog/2020-outreach-partners.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+* Spread the word about our call for proposals for talks, workshops & Dev sprint and help us find speakers, mentors, and reviewers from your community for the conference.
+
+## What you can expect from us
+
+For supporting the conference, our outreach partners can expect following in return from PyCon India:
+
+* We will display your Logos and Names on the conference website. 
+* We will also promote your community on our social media channels as a supporter of the conference.
+
+If you are an organization or a community interested in becoming an outreach partner for PyCon India 2020, please reach out to us at [link]
```
Comment (vipulgupta2048) [inline content/blog/2020-outreach-partners.md:29]: @anistark please suggest the [link]

File: content/blog/2020-outreach-partners.md (hunk 1 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-outreach-partners.md:7]: I think the single sentence looks better. Also, how about `creating a more inclusive, accessible conference this year`

File: content/blog/2020-outreach-partners.md (hunk 4 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-outreach-partners.md:17]:
```suggestion
We would like to invite your community to be a Outreach partner for PyCon India 2020. You can help us spread the word about the conference among your members who are looking to learn something regarding Python. We expect our outreach partners to help us in the following ways: 
```
?

**Comment by Sc0rpi0n101** [comment]
It's a supplement to the email(https://docs.google.com/document/d/1ulsyKBGBm4CecQPENRBmafqdFFags3ZS1GEOYTHFH0E/edit?usp=sharing). 

If some community is interested to be an outreach partner after seeing a social media post or something, not getting the mail. And they need the information what, why, how, etc. That's where it will be useful.

File: content/blog/2020-outreach-partners.md (hunk 6 above)
Comment (anistark) [reply content/blog/2020-outreach-partners.md:29]: @abhishekmishragithub @palnabarun Do we've an email for this? like partnership(s) [at] .... ??

Comment (abhishekmishragithub) [reply content/blog/2020-outreach-partners.md:29]: @anistark we don't have an email id for it. We can use contact@in.pycon.org  (from last year)

Comment (anistark) [reply content/blog/2020-outreach-partners.md:29]: Who gets contact emails? @abhishekmishragithub

Comment (abhishekmishragithub) [reply content/blog/2020-outreach-partners.md:29]: We need to add people to alias, @palnabarun or @sayanchowdhury can help.
It was already set up last year by Vijay (@bravegnu)

Comment (anistark) [reply content/blog/2020-outreach-partners.md:29]: Till then, let's use a google form or something? Less dependency.

Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:29]: +1 more control, transparency, and lower dependency on just one person. 
@anistark maybe we can wait for comments or do provide the form link

File: content/blog/2020-outreach-partners.md (hunk 4 above)
Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:17]: something regarding --> more regarding

File: content/blog/2020-outreach-partners.md (hunk 2 above)
Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:11]: @anistark :up:

File: content/blog/2020-outreach-partners.md (hunk 1 above)
Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:7]: Works

File: content/blog/2020-outreach-partners.md (hunk 2 above)
Comment (Sc0rpi0n101) [reply content/blog/2020-outreach-partners.md]: I have updated this in the content. If there is something wrong about the stats and need to be changed, we can change it later.

File: content/blog/2020-outreach-partners.md (hunk 6 above)
Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md:27]: @anistark @abhishekmishragithub Need the form link folks and finish this PR.

File: content/blog/2020-outreach-partners.md (hunk 2 above)
Comment (vipulgupta2048) [reply content/blog/2020-outreach-partners.md]: Sounds good, I will still ping @anistark again and then resolve the last pending comments.

**Comment by Sc0rpi0n101** [comment]
I've added the form link to the blog, but the blog still contains the About Page link, which has not been deployed yet.

**Pull request merged**